### ISSUE TITLE
[Request] Disable looping on extracted audio files

### ISSUE THREAD

**Issue by garfielfing** [vgmstream/vgmstream#1531]
A longstanding issue that's been reported is that VGMStream, used with Foobar, causes some audio files to loop when converted. This is most noticeable with video game SFX, not music, such as creature sounds or weapon swings, and one example is Dark Souls. When converted, audio like this will often loop multiple times before fading out. While this isn't necessarily a problem and is fine as a feature, it'd be convenient to have an option to disable looping entirely. 

For example, currently, if you wanted to extract the sound of a dragon roaring, which in this case will get looped multiple times then fade out, do you then have to use Audacity or a similar program to cut out the loops + fadeout so you can have one single clean take of the sound effect you want.

Basically, I think there should be an option, when converting audio, to disable looping when the files are exported.

Thank you all for your awesome work!

**Comment by EdnessP** [comment]
Looping can be disabled in Foobar under Library -> Configure -> Playback -> Decoding -> vgmstream.

**Comment by garfielfing** [comment]
Never knew about that. Thank you!

**Comment by garfielfing** [comment]
-
